taskqueue/deferred/pom.xml

```diff
@@ -32,7 +32,7 @@
         <maven.compiler.target>1.7</maven.compiler.target>
         <maven.compiler.source>1.7</maven.compiler.source>
         <codehaus-versions-maven-plugin-version>2.3</codehaus-versions-maven-plugin-version>
-        <maven-war-plugin-version>3.0.0</maven-war-plugin-version>
+        <maven-war-plugin-version>2.6</maven-war-plugin-version>
```

Comment:
Strange, the taskqueue sample is the only one where it tried to downgrade the war plugin. There are plenty of other samples that it didn't touch. https://github.com/GoogleCloudPlatform/java-docs-samples/search?p=2&q=maven-war-plugin-version&type=Code&utf8=%E2%9C%93